drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -455,6 +461,10 @@ static int stm32_clock_control_get_subsys_rate(const struct device *clock,
 	case STM32_CLOCK_BUS_AHB2:
 	case STM32_CLOCK_BUS_AHB3:
 	case STM32_CLOCK_BUS_AHB4:
+#if defined(CONFIG_SOC_SERIES_STM32H7RSX)
+	/* HCLK5 i a possible source clock for some peripherals */
+	case STM32_SRC_HCLK5:
```

Comment:
STM32_SRC_HCLK1/2/3/4 (added in the previous commit) and STM32_CLOCK_BUS_AHB5 should also be added here.

Reply:
moved to the other commit

Reply:
STM32_SRC_HCLK1/2/3/4 and STM32_CLOCK_BUS_AHB5 cases are still missing in this function.